contributor/01-introduction-article.md

```diff
@@ -0,0 +1,97 @@
+You can also watch the videos ([introduction](intro-trusted-committer),[lowering the barriers to entry](tc-lowering-the-barriers-to-entry)) instead of the articles.
+
+### Trusted Committer
+
+A Trusted Committer will be your host for your stay in the hosting community.
+They are the gatekeepers to the project's code repository and will move your contribution closer to production once they accepted it. 
+It is their role to mentor you on your way to contributing to their community, e.g. by providing good information to enable you to help yourself or help you in other cases.
+Such information could be established house rules for reviews, proposal of larger changes,  pointers to documentation or code sections relevant for your contribution.
+
+They also need to care about product quality, sustainability and project evolution from a technical as well as general perspective, about reducing the barrier to making to contributions for everyone as well as about caring for their community in general.
+Caring for the community involves keeping it healthy, upleveling it and its participants and advocating its needs in their organization.
+
+
+### Product Owner
+[EDITOR NOTE: We actually have no defined articles here. Below rough draft is based upon https://github.com/InnerSourceCommons/InnerSourceLearningPath/pull/96/files#diff-1fc7c3a09a5ec014a5ba5313dc6473fa]
+The role of the Product Owner has some similarity with the product owner role of your average project.
+However, there are some differences - depending on the size of the project, this role is often filled by the same person that acts as trusted committer.
+In larger projects or in teams that only partially use InnerSource as an approach to fill their needs by accepting contributions this role is likely filled by a person separate from a trusted committer.
+
+Your interaction with this role will likely focus on working out the fit of your contribution into the general product and its road map. 
+You might work this role to make sure that general aspects of documentation or consistency of UI/UX are kept upon merging your contribution.  
+
+Last but not least someone acting as product owner might have been involved in bringing the project, its benefits and community to your attention. 
+
+
+If you want to learn in more detail what these other roles are about, and we encourage you to do so, we've prepared sections about the [Trusted Committer](link) and the [Product Owner](link) too.
+
+
+## Overview of this section and its segments
+In the following 5 videos you will learn more in detail about the various aspects introduced before. 
+
+The second segment, i.e. the one after this segment, will detail _mindset and habit_ that create opportunities to _become an InnerSource Contributor_.
+
+In the third segment we'll look into the _ethos of the Contributor_ - i.e. aspects of _behavior_ that will lead to a pleasant and productive time for you and the home team and might spark more collaboration.
```

Comment:
Do we use Home team? Shouldn't that be host team?

Reply:
Yes, you're right.

```suggestion
In the third segment we'll look into the _ethos of the Contributor_ - i.e. aspects of _behavior_ that will lead to a pleasant and productive time for you and the host team and might spark more collaboration.
```

Reply:
#112 fixed that as well.

Reply:
True